subsys/dfu/include/dfu_ctx_mcuboot.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/** @file dfu_ctx_mcuboot.h
+ *
+ * @defgroup dfu_ctx_mcuboot MCUBoot DFU Context Handler
+ * @{
+ * @brief DFU Context Handler for updates performed by MCUBoot
+ */
+
+#ifndef DFU_CTX_MCUBOOT_H__
+#define DFU_CTX_MCUBOOT_H__
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <dfu/dfu_context_handler.h>
```

Comment:
I don't see how this header depends on this other header, I believe this include should be removed.

Reply:
It's needed for `struct dfu_ctx dfu_ctx_mcuboot` to get the `struct dfu_ctx`

Reply:
Ah, I missed that, sorry.